stock_picking_mass_assign/data/cron_data.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="ir_cron_check_assign_all" model="ir.cron">
+        <field name="name">Check Availability of Delivery Orders</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">hours</field>
+        <field name="numbercall">-1</field>
+        <field eval="False" name="doall"/>
```

Comment:
[Please put name attribute before the value](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#format)